GETTING_STARTED.md

```diff
@@ -0,0 +1,380 @@
+
+```
+{
+  "page": { // result can be paged with two query parameters offset and limit
+    "offset": null,
+    "limit": null,
+    "totalSize": 18 // total size of this table
+  },
+  "rows": [
+    { // row object
+      "id": 24, // Unique column ID
+      "values": [
+        {
+          "de": "Schweden",
+          "es": "Suecia",
+          "fr": "Suède",
+          "it": "Svezia"
+        },
+        "2nd value"
+      ]
+    },
+    {
+      "id": 25,
+      "values": [
+        {
+          "de": "Deutschland",
+          "en": "Germany"
+        },
+        "2nd value"
+      ]
+    }
+    // ... more rows to come
+  ]
+}
+```
+
+### Retrieving a specific column
+
+`GET /tables/:tableid/columns/:columnid`
+
+Call this endpoint to retrieve the definition of one specific column. Same column object definition as above. 
+
+```
+{
+  "id": 1,
+  "ordering": 1,
+  "name": "name",
+  "kind": "shorttext"
+  // ...
+}
+```
+
+### Retrieving a specific row
+
+`GET /tables/rows/:rowid`
+
+Call this endpoint to retrieve a specific row. Same row object definition as above.
+
+```
+{
+  "id": 24,
+  "values": [
+    {
+      "de": "Schweden",
+      "es": "Suecia",
+      "fr": "Suède",
+      "it": "Svezia"
+    },
+    "2nd value"
+  ]
+}
+```
+
+### Retrieving a specific cell
+
+`GET /tables/:tableid/columns/:columnid/rows/:rowid`
+
+Call this endpoint to retrieve a single value of a specific cell. Value object can be different for the various data types.
+
+```
+{
+  "value": {
+    "de": "Schweden",
+    "es": "Suecia",
+    "fr": "Suède",
+    "it": "Svezia"
+  }
+}
+```
+
+## 2.2. Data types & column kinds
+
+Each column has a specific data type defined in its `kind` field. There are a number of different data types which will be described in this chapter.
+
+### Primitive data types
+
+Currently there are a few primitive data types. All primitive data types can be used in a multi-language column or a language neutral column.
+
+* `text`, `shorttext` and `richtext`
+* `numeric` and `currency`
+* `date` and `datetime`
+* `boolean`
+
+#### Examples 
+
+For text there are three different column kinds. `text`, `shorttext`, and `richtext`. All three are syntactically the same but semantically different.
+
+* `shorttext` should only contain a word or a short sentence
+* `text` is meant for texts without formatting
+* `richtext` is meant for text with markdown syntax
+
+```
+// shorttext example
+"This is a text"
+```
+
+The data types `numeric` and `currency` are meant for storing numerical values like integers and floats. `currency` can be used to store prices. It is often combined with `"languageType": "country"` as most prices are country specific.
+
+```
+// numeric example
+1337.42
+```
+
+To store date and time information the data types `date`, and `datetime` can be used. 
```

Comment:
Output and accepted input is strings, it makes sense to store ISO-type date strings but the backend is indefferent about that.